xml/admin_administration.xml

```diff
@@ -49,19 +49,39 @@
     </para>
    </listitem>
   </itemizedlist>
-
+  
   <para>
-   Transactional updates are managed by the
-   <command>transactional-update</command> script, which is called once a day.
-   The script checks if any updates are available. If there are any updates to
-   be applied, a new snapshot of the root file system is created and the system
-   is updated using the <command>zypper dup</command>. All updates released to
-   this point are applied. The snapshot is then marked as active and will be
-   used after the next reboot of the system. Ensure that the cluster is
-   rebooted as soon as possible after the update installation is complete,
-   otherwise all changes will be lost.
+   Transactional updates are managed by the <command>transactional-update</command> 
+   script, which is called once a day. The script checks if any updates are 
+   available. If there are any updates to be applied, a new snapshot of the root 
+   file system is created in the background and is updated using 
+   <command>zypper dup</command>. All updates released to this point are applied.
+   The running filesystem/machine state is left untouched.
   </para>
-
+  <para>
+   The new snapshot, once completely updated, is then marked as active and will 
+   be used as the new default after the next refresh of the system. Only the last
```

Comment:
/refresh/reboot/
I doubt that many people will understand that refresh in this context means reboot.